packages/core/src/types/tasks.ts

```diff
@@ -21,17 +22,16 @@ export interface Task {
   run: () => Promise<TaskResult>;
 }
 
-export interface TaskContext {
-  readonly cliArguments: string[];
-  readonly cliFlags: RunFlags;
-  readonly parsers: Map<ParserName, CreateParser<ParserOptions, Parser<ParserReport>>>;
-  readonly config: CheckupConfig;
-  readonly pkg: PackageJson;
-  readonly paths: FilePathsArray;
+export interface Action {
+  name: string;
+  isEnabled: () => boolean;
+  threshold: number;
+  value: number;
+  message: () => string;
```

Comment:
We should make this a getter.

```suggestion
  readonly message: string;
```